data/json/itemgroups/main.json

```diff
@@ -0,0 +1,227 @@
+      ["raw_leather", 0],
+      ["rebreather_on", 0],
+      ["rebreather_xl_on", 0],
+      ["rep_crossbow", 0],
+      ["rm13_armor_on", 0],
+      ["rock", 0],
+      ["shishkebab_on", 0],
+      ["shocktonfa_on", 0],
+      ["smokebomb_act", 0],
+      ["spaghetti_cooked", 0],
+      ["stick", 0],
+      ["straw_pile", 0],
+      ["survnote", 0],
+      ["testflames", 0],
+      ["thermal_gloves_on", 0],
+      ["thermal_mask_on", 0],
+      ["thermal_outfit_on", 0],
+      ["thermal_socks_on", 0],
+      ["thermal_suit_on", 0],
+      ["tinfoil_hat", 0],
+      ["toolset", 0],
+      ["torch_lit", 0],
+      ["trimmer_on", 0],
+      ["UPS_on", 0],
+      ["vaccine_shot", 0],
+      ["veggy", 0],
+      ["veggy_cooked", 0],
+      ["veggy_tainted", 0],
+      ["v_table", 0],
+      ["wearable_light_on", 0],
+      ["wrapper", 0]
+    ]
+  },{
+    "type" : "item_group",
+    "id" : "rare",
+    "//" : "Items that should have very low frequencies and or few locations.",
+    "items":[
+      ["12mm", 20],
+      ["adrenaline_injector", 20],
+      ["ampoule", 100],
+      ["antifungal", 10],
+      ["antiparasitic", 20],
+      ["armor_chitin", 10],
+      ["bio_blade", 10],
+      ["bio_speed", 10],
+      ["bot_c4_hack", 30],
+      ["bot_laserturret", 20],
+      ["bot_mininuke_hack", 10],
+      ["c4", 50],
+      ["canister_goo", 80],
+      ["cleansuit", 100],
+      ["conversion_battle", 10],
+      ["conversion_sniper", 10],
+      ["dump_pouch", 100],
+      ["electrohack", 30],
+      ["fetus", 10],
+      ["ftk93", 10],
+      ["hazmat_suit", 50],
+      ["hk_g80", 20],
+      ["id_military", 30],
+      ["id_science", 20],
+      ["knife_rm42", 20],
+      ["l-stick", 10],
+      ["large_repairkit", 20],
+      ["laser_pack", 100],
+      ["laser_rifle", 10],
+      ["m240", 10],
+      ["m249", 10],
+      ["m60", 10],
+      ["mask_bal", 100],
+      ["militarymap", 20],
+      ["mininuke", 10],
+      ["mutagen", 80],
+      ["nx17", 10],
+      ["optical_cloak", 20],
+      ["pickaxe", 10],
+      ["plasma", 80],
+      ["plasma_rifle", 10],
+      ["plut_cell", 100],
+      ["portal", 20],
+      ["power_armor_basic", 50],
+      ["power_armor_frame", 40],
+      ["power_armor_heavy", 30],
+      ["power_armor_helmet_basic", 60],
+      ["power_armor_helmet_heavy", 30],
+      ["power_armor_helmet_light", 30],
+      ["power_armor_light", 30],
+      ["processor", 150],
+      ["purifier", 120],
+      ["pur_tablets", 200],
+      ["recipe_medicalmut", 100],
+      ["remotevehcontrol", 80],
+      ["rm11b_sniper_rifle", 10],
+      ["rm13_armor", 20],
+      ["rm51_assault_rifle", 10],
+      ["royal_jelly", 80],
+      ["RPG", 10],
+      ["RPG-7_ammo", 10],
+      ["rx11_stimpack", 50],
+      ["rx12_injector", 50],
+      ["stimpack_ammo", 100],
+      ["survivormap", 50],
+      ["survnote", 10],
+      ["tool_rdx_charge", 10],
+      ["v29", 10]
+    ]
+  },{
+    "type" : "item_group",
+    "id" : "player_crafted",
```

Comment:
It's like the rare group. It's just an FYI to remind people not to place these items anywhere. It's not important though.  
I can either remove it or make the comment more explicit.